docs/assets/stylesheets/components/_documentation_tabs.scss

```diff
@@ -0,0 +1,281 @@
+.app-navigation {
```

Comment:
The only thing I'm not a great fan of is this being app-_navigation_ 

I can see what we're trying to achieve, and that we're basically overlaying the navigation style over it, but it still feels a bit off.  I realise that the class `.app-tabs` is already taken for the examples... but I do sort of feel that this should be its own thing (or "block" I suppose in BEM terms)

We have 2 different tabs components:
 - example tabs
 - documentation/content tabs

To my mind they both have the same behaviours so should have the same js `data-module` (they currently don't - the example tabs use our own tabs js and the content ones use govuk), and then all the styles for them should be wrapped up into a block level class for that style of tabs. e.g. `.app-example-tabs` and `.app-content-tabs` as I don't imagine there are many shared styles so they're not really a variant.

What are your thoughts?

Reply:
Yes I do understand and agree. Although they behave slightly different - the document/content tabs turn into anchor links on mobile screens whilst the example tabs remain tabs. I think the idea was to utilise the tab component for functionality but we could build that logic separately.